sqldb/sqlc/migrations/000002_amp_invoices.up.sql

```diff
@@ -10,26 +10,35 @@ CREATE TABLE IF NOT EXISTS amp_invoice_payments (
     -- Timestamp of when the first htlc for this payment was accepted.
     created_at TIMESTAMP NOT NULL,
 
-    -- If settled, the invoice payment related to this set id.
-    settled_index INTEGER REFERENCES invoice_payments(id),
+    -- When the invoice was settled.
+    settled_at TIMESTAMP,
```

Comment:
Same above re the event sourced angle, applies below to `settled_index` as well.